src/containers/Settings.jsx

```diff
@@ -217,6 +284,7 @@ class Settings extends React.Component {
           </CardActions>
         </Card>
         <div>{this.renderTextingHoursForm()}</div>
+        <div>{this.renderTwilioAuthForm()}</div>
```

Comment:
Can we condition this on an global environment variable allowing this?

Reply:
I actually don't see an easy way to do that. Global variables don't seem to be available in the component, so then the only way would be to load that with the Organization. Although I could be missing something.

Reply:
nevermind, I figured it out.